### ISSUE TITLE
[M0rehud Classic] Casual overlay broken in 1440p

### ISSUE THREAD

**Issue by georgiesgithub** [Hypnootize/m0rehud#63]
![image](https://github.com/user-attachments/assets/692d318d-6cee-4eb7-8637-bc2b8f658911)
Casual Button does not fully cover screen in 2560x1440, 16:9 on m0rehud classic.

**Comment by Hypnootize** [comment]
Yeah that's mostly because the default footer is missing but seeing how it's intended to be a very barebones HUD, menu wise, I've never really cared too much about fixing it. You could use the one frorm the main version by importing `MatchMakingPlayList.res` `MainMenuPlaylistEntry.res` and `MatchMakingDashboardSidePanel.res` to fix that.
